### DISCUSSION TITLE
Why is postgres:14.2 promoted as default?

### DISCUSSION THREAD

**Discussion by Blackskyliner** [coder/coder#13438] [General] [answered]
As I did not found anything about this. 
The Docker image of 14.2 is already 2 years old. 

Why is especially this version used? 
Are there any incompatibilities using a more recent version of postgres (e.g. 14.12)?

**Comment by matifali** [comment]
I use Postgres 16 on a deployment and have never seen any issues.
@kylecarbs or @ammario can help us out here.

**Reply by mattos-filipe** [reply]
Same here

**Comment by johnstcn** [comment, marked as answer]
I assume you're referring to the hard-coded version in our `docker-compose.yaml`.
There isn't any particular reason that 14.2 is promoted as default; it was probably the "latest" version when that file was added. I'll open a separate PR to bump that. 
Anecdotally, I also run my personal deployment on postgres 16.2 and have not run into any issues.
